frontend/src/components/GListItemContent.vue

```diff
@@ -19,6 +19,7 @@ SPDX-License-Identifier: Apache-2.0
   <div
     v-if="slots.description || props.description"
     class="text-body-2 text-high-emphasis pt-1"
+    :class="descriptionClass"
```

Comment:
alternatively have a dense prop to control the padding
also, why has the description text-high-emphasis and not text-medium-emphasis? cc @holgerkoser

Reply:
bug